projects/swimlane/ngx-ui/src/lib/components/notification/notification.component.ts

```diff
@@ -35,12 +39,18 @@ export class NotificationComponent {
   @Input() showClose: boolean;
   @Input() timestamp: any;
   @Input() icon: string;
+  @Input() timeout: false | number;
 
   @Output() close = new EventEmitter();
   @Output() pause = new EventEmitter();
   @Output() resume = new EventEmitter();
 
-  timeout: any;
+  get animationDuration() {
+    if (this.timeout === false) {
```

Comment:
do we want a falsy check here? or should we default `timeout` to false?

Reply:
ahh yeah, yeah let's go with that I think, so it handles the `undefined` case and lets 0 through